scripts/run_e2e_tests.sh

```diff
@@ -46,7 +46,7 @@ function main() {
 
   trap cleanup EXIT
 
-  kubectl proxy &
+  kubectl proxy &>/dev/null &
```

Comment:
Why silence the output, couldn't it help if the test borks?

Reply:
So the annoying this was this would interleave the proxy output with the test and it was confusing what was coming from where, but I guess that's better than nothing so will re-add

Reply:
You could consider something like: 
`kubectl proxy |sed -e 's/^/kube proxy: /'&`